src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -76,6 +78,9 @@
 
 class SerializerTest extends TestCase
 {
+    /**
+     * @legacy
```

Comment:
Is it really legacy?

Reply:
I have to mark the test as legacy because it will generate a deprecation warning. 

Ie it runs `new Serializer([new \stdClass()]);` which is deprecated.